e2e/ctl_v3_member_test.go

```diff
@@ -83,10 +83,7 @@ func memberRemoveTest(cx ctlCtx) {
 		cx.t.Fatalf("expected %d, got %d", n1, len(resp.Members))
 	}
```

Comment:
The checks performed by line 74 to 84 is already done by `memberToRemove()`. Remove those?